### PR TITLE
msamprz/binary-static-professional-cta

### PR DESCRIPTION
**Pull request by msamprz**

### CODE REVIEW

File: src/javascript/app/pages/trade/purchase.js [hunk 1]
```diff
@@ -104,18 +106,41 @@ const Purchase = (() => {
...
-                        if (/FinancialBinaries/.test(error.code)) {
-                            additional_message = localize('Try our [_1]Volatility Indices[_2].', [`<a href="${urlFor('get-started/binary-options', 'anchor=volatility-indices#range-of-markets')}" >`, '</a>']);
-                        } else if (/Random/.test(error.code)) {
-                            additional_message = localize('Try our other markets.');
+                    BinarySocket.wait('get_account_status').then(response => {
+                        confirmation_error.setVisibility(1);
+                        let message = error.message;
+                        if (/NoMFProfessionalClient/.test(error.code)) {
+                            const has_professional_requested = (getPropertyValue(response, ['get_account_status', 'status']) || []).includes('professional_requested');
+                            const has_professional_rejected  = (getPropertyValue(response, ['get_account_status', 'status']) || []).includes('professional_rejected');
```
Comment (negarn) [inline src/javascript/app/pages/trade/purchase.js]: why not store `getPropertyValue(response, ['get_account_status', 'status']) || []` and reuse it?

File: src/javascript/app/pages/trade/purchase.js [hunk 2]
```diff
@@ -104,18 +106,41 @@ const Purchase = (() => {
...
+                            if (has_professional_requested) {
+                                message = localize('Your application to be treated as a professional client is being processed.');
+                            } else if (has_professional_rejected) {
+                                message = `${localize('Your request to be treated as a professional client is not approved.')}&nbsp;${localize('Please check your inbox for more details.')}<br /><br />${localize('Your account remains under the retail client category. You are welcome to reapply as a professional client at any time.')}`;
```
Comment (negarn) [inline src/javascript/app/pages/trade/purchase.js]: there should also be a button to professional page, similar to the else

Comment (msamprz) [reply src/javascript/app/pages/trade/purchase.js]: Noted and addressed.

File: src/javascript/app/pages/trade/purchase.js (hunk 1 above)
Comment (msamprz) [reply src/javascript/app/pages/trade/purchase.js]: Good idea, addressed in latest commit.

**Pull request merged**